src/js/construct-element.js

```diff
@@ -23,19 +28,28 @@ export default {
 		let contentHTML;
 		if (opts.content.detail) {
 			contentHTML = `
-				<div class="${opts.messageClass}__content--header">
```

Comment:
I would argue that it's more readable to have all of the markup in each conditional, instead of adding bits as you go.

Reply:
Done!